docs/tutorial.md

```diff
@@ -394,7 +394,107 @@ What's going on here? Let's step through the code.
 
 (sec_extracting_individuals)=
 
-## Extracting particular SLiM individuals
+## Extracting SLiM individuals
+
+### Extracting a sample of individuals
+
+Often we would like to sample a number of individuals from a population.
+This section provides an example workflow and draws
+attention to some easy-to-miss details for this approach.
+
+Let's start with a nonWF simulation similar to what we've seen previously.
```

Comment:
Is this the *same* as in `example_sim.slim`?

Reply:
Yes. So, should delete the simulation step